scripts/directtools/__init__.py

```diff
@@ -709,11 +721,12 @@ def plotSofQW(workspace, QMin=0., QMax=None, EMin=None, EMax=None, VMin=0., VMax
     :type colorscale: str
     :returns: a tuple of (:mod:`matplotlib.Figure`, :mod:`matplotlib.Axes`)
     """
-    # Accept both workspace names and actual workspaces.
     workspace = _normws(workspace)
     if not _validate._isSofQW(workspace):
         logger.warning("The workspace '{}' does not look like proper S(Q,W) data. Trying to plot nonetheless.".format(str(workspace)))
-    isSusceptibility = workspace.YUnit() == 'Dynamic susceptibility'
+    qHorizontal = workspace.getAxis(0).getUnit().name() == 'q'
+    isSusceptibility = workspace.YUnitLabel() == 'Dynamic susceptibility'
+    print(workspace.YUnit())
```

Comment:
Is this `print` statement required or was it left here for debugging purposes?

Reply:
Nice catch! It is some debuggign leftover.